admin/syslog-ng/Makefile

```diff
@@ -85,7 +85,8 @@ define Package/syslog-ng/install
 	$(INSTALL_DATA) ./files/syslog-ng.conf $(1)/etc
 	touch $(1)/etc/syslog-ng.d/.keep
 
-	$(INSTALL_BIN) ./files/logread $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/sbin
+	$(INSTALL_BIN) ./files/logread $(1)/sbin
```

Comment:
Sounds like it should be fixed in LuCI.

Reply:
LuCI uses an absolute path, which can be improved. But I think, this should be fixed in the Makefile of syslog-ng. It was using a non-standard location for logread and this is described in the commit message and as well in a pull request.

Reply:
We disagree on this point. Linux PATH uses multiple directories for a reason.

Anyway, feel free to merge. I don't have a strong opinion on this.

Reply:
I think we understand each other. I don't like hard coding as well, but in the first place it should be fixed in LuCI. Until then it must be fixed here, because it breaks LuCI functionality to view system and kernel log.